styles/modal.css

```
@@ -79,10 +98,176 @@ body.modal-open {
+  }
+
+  .plot-container h3 {
+    text-align: center;
+  }
+  
+  .plot-container .detail {
+    text-align: left;
+  }
+}
+
+@keyframes slideUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)